README.md

```diff
@@ -184,10 +184,12 @@ Test and commit the new translations. There exists schemes for testing other lan
 
 #### Via CLI
 
-1. Add the new strings to the English source file `Client/Ecosia/L10N/en.lproj/Ecosia.strings`
-2. Push it to Transifex
+1. Pull the source file
+2. Add the new strings to the English source file `Client/Ecosia/L10N/en.lproj/Ecosia.strings`
+3. Push it to Transifex
 
 ```bash
+tx pull -s
```

Comment:
isn't is `tx pull -af` ?

Reply:
yes true, but you miss the source file (en) when you don't append `-s` . So to be super accurate it should be `tx pull -afs` when you want to force all the changes from remote

Reply:
@ecotopian  Then I think it would make sense to update the documentation to `tx pull -afs`